pinot-common/src/main/java/com/linkedin/pinot/common/data/StarTreeIndexSpec.java

```diff
@@ -40,9 +41,20 @@
 
   private boolean _excludeSkipMaterializationDimensionsForStarTreeIndex;
 
+  private static final String SEPARATOR = ",";
+  private static final String DIMENSIONS_SPLIT_ORDER_PARAM = "dimensionsSplitOrder";
```

Comment:
do we need all these constants if we are using jackson?

Reply:
thanks, done